README.md

```diff
@@ -97,20 +97,19 @@ first -- the `configure` script will download it for you.
 
 There are two distributions of ACE/TAO that can be used with OpenDDS:
 
-* OCI ACE/TAO 2.2a patch 19 or later
+* OCI ACE/TAO 2.2a patch 20 or later
   * This will be automatically downloaded by default when using the configure
     script.
   * Can be manually downloaded from:
     * http://download.objectcomputing.com/TAO-2.2a_patches/
-* DOC Group ACE 6.5.12 / TAO 2.5.12 or later in the ACE 6.x / TAO 2.x series
+* DOC Group ACE 6.5.13 / TAO 2.5.13 or later in the ACE 6.x / TAO 2.x series
```

Comment:
Shouldn't this be extended with ACE 7.0.2 / TAO 3.0.2?

Reply:
There isn't the level of testing across multiple platforms on ACE 7 that would lead us to call it "supported."  It's good that we're minimally testing it in CI so we don't accidentally introduce some incompatibility, but this is essentially for maintainers and not for end users.